src/app/series/directives/series-podcast.component.css

```diff
@@ -25,3 +25,6 @@
 .feed-url button:hover, .feed-url button:focus {
   background-color: #e28b42;
 }
+input[type=checkbox] {
+  vertical-align: middle;
+}
```

Comment:
+1 for styleguide. think we need this on the episode podcast.component.css too! 
![image](https://cloud.githubusercontent.com/assets/16007382/24929354/a677e6c0-1ed3-11e7-949b-9a2e68196d1f.png)